schemas/compat-data-schema.md

```diff
@@ -249,6 +249,26 @@ Examples:
 }
 ```
 
+### Initial versions
+
+The following table indicates initial versions for browsers in BCD. These are the earliest possible version numbers allowed to be used. These are not naturally "1" or "1.0" as explained in the notes column.
+
+| Browser          | Initial version | Notes                                                                                                                      |
+| ---------------- | --------------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Chrome           | 1               |                                                                                                                            |
+| Chrome Android   | 18              | First stable release started at 18. No Chrome Android 17 or earlier was ever released.                                     |
+| Edge             | 12              | EdgeHTML, continuation of IE 11 (another initial version is Edge 79 when Edge switched to Chromium).                       |
```

Comment:
suggestion: this is kinda complicated and I think being wordier here might help?

```suggestion
| Edge             | 12              | EdgeHTML versioning started at 12, continuing from Internet Explorer 11. After version 18, Edge jumped to version 79, synchronizing with the Chromium versioning scheme. |
```